model/src/main/kotlin/Provenance.kt

```diff
@@ -60,7 +60,8 @@ data class ArtifactProvenance(
  */
 data class RepositoryProvenance(
     /**
-     * The VCS repository that was downloaded.
+     * The VCS information that was passed to the downloader. This potentially still contains an unresolved "moving"
+     * VCS revision, like "master" for Git.
```

Comment:
Maybe "The VCS information that was used to resolve this provenance. ..." as it is a bit more precise. In the same sense the comment for `resolvedRevision` could be changed to say "The resolved VCS revision of the source code."

Reply:
I've chosen a slightly different wording as I didn't want to say "resolve this provenance" because it's not the provenance itself that's being resolved, but provenance is the result of resolution.